rdopkg/utils/specfile.py

```diff
@@ -245,6 +245,11 @@ def patches_apply_method(self):
             return 'autosetup'
         return 'rpm'
 
+    def set_commit_ref_macro(self, ref):
+        self._txt = re.sub(
+            r'\%global commit \w+',
```

Comment:
I'd go with `r'^\%global...'` and `flags=re.M` to prevent changing commented lines etc.